File: Programmers/Lv0/자릿수더하기.cpp
#include <string>
#include <vector>
using namespace std;

int solution(int n) {
    int answer = 0;
    while (n > 0)
    {
        answer += n % 10;
        n /= 10;
    }
    return answer;
}

// 파이썬의 방식처럼 문자열로 바꾸어 자릿수를 추출하여 더함
int solution(int n) {
    int answer = 0;
    string s = to_string(n);
    for(char c : s)
        answer += (c - '0');
    return answer;
}

File: Programmers/Lv0/자릿수더하기.py
def solution(n):
    answer = 0
    n = str(n)
    for i in range(len(n)):
        answer += int(n[i])
    return answer

# map을통해 문자열을 정수로 바꾸어 리스트화 하고 자릿수를 더하는 방법
def solution(n):
    answer = sum(list(map(int,str(n))))
    return answer